src/shared/Table/TableView.tsx

```diff
@@ -239,18 +239,18 @@ export const TableView = <TRow, TCol>({
           <OverflowMenu breakpoint={breakpoint}>
             <OverflowMenuContent isPersistent>
               <OverflowMenuGroup isPersistent groupType="button">
-                <OverflowMenuItem>
-                  {actions &&
-                    actions.map((a, idx) => (
+                {actions &&
+                  actions.map((a, idx) => (
+                    <OverflowMenuItem>
```

Comment:
This needs the `key` that was in the button before

Reply:
of course, right fixed